Mono.Cecil.Cil/PortablePdb.cs

```diff
@@ -249,6 +249,12 @@ public void Dispose ()
 				return;
 
 			WritePdbHeap ();
+
+			if (module_metadata.string_heap != pdb_metadata.string_heap) {
```

Comment:
My preference is to be defensive and keep the check or at least have an assert but I can remove it if you insist.